src/locales/en_US.ts

```diff
@@ -785,6 +785,12 @@ const strings = {
   edge_login_fail_message: 'Could not connect to network. Please check your network connection and try again.',
   edge_login_fetching: 'Fetching Edge Login info...',
   modal_addressexplorer_message: 'Show Address in Block Explorer?',
+  can_be_partial_quote: 'Maybe Partial Quote',
```

Comment:
The string name should have `_title` on the end, per a previous sidebar. I also think we should drop the word "Maybe", since that's included in the explanation below.

Reply:
This was also to be consistent with `estimated_exchange_rate` but that's the a fair catch. I'll change to adhere to new standards